src/main/java/com/artipie/RepoConfig.java

```diff
@@ -191,6 +211,15 @@ public Permissions permissions() {
             .thenApply(yaml -> new RepoConfig(storages, prefix, yaml));
     }
 
+    /**
+     * Repo part of YAML.
+     *
+     * @return Async YAML mapping
+     */
+    public YamlMapping repoConfig() {
+        return Objects.requireNonNull(this.yaml.yamlMapping("repo"), "yaml repo is null");
```

Comment:
@olegmoz I see that this method was just made public, but I think that it would be a good idea to change its message to something like `yaml repo config is null or absent`; I think that it represents better the error, and it is important now that we have this method public which can be called by anyone since it is public. WDYT?